src/metabase/notification/send.clj

```diff
@@ -147,20 +147,24 @@
             (do
               (log/debugf "[Notification %d] Found %d handlers" notification-id (count handlers))
               (doseq [handler handlers]
-                (let [channel-type (:channel_type handler)
-                      messages     (channel/render-notification
-                                    channel-type
-                                    notification-payload
-                                    (:template handler)
-                                    (:recipients handler))]
-                  (log/debugf "[Notification %d] Got %d messages for channel %s with template %d"
-                              notification-id (count messages)
-                              (handler->channel-name handler)
-                              (-> handler :template :id))
-                  (doseq [message messages]
-                    (log/infof "[Notification %d] Sending message to channel %s"
-                               notification-id (:channel_type handler))
-                    (channel-send-retrying! notification-id (:payload_type hydrated-notification) handler message))))
+                (try
```

Comment:
fix an use case here: the idea is if a handler is failed to render/send, the other handler shouldn't be failed.

see `partial-channel-failure-will-deliver-all-that-success-test` test